DigitalExcellence/dex-frontend
# Feature/298 seo friendly urls

## Description

The creation of user friendly URLs so that the GoogleBot ranks the pages higher

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Go to project detail page
2. Check if url consists of the project ID + project title

## Link to issue

Closes: #298
